cocotb/share/include/exports.h

```diff
@@ -5,11 +5,12 @@
 #ifndef COCOTB_EXPORTS_H_
 #define COCOTB_EXPORTS_H_
 
-// Make cocotb work correctly when the default visibility is changed to hidden
-// Changing the default visibility to hidden has the advantage of significantly reducing the code size,
-// load times as well as letting the optimizer produce better code
+// Make cocotb work correctly when the default visibility is changed to hidden.
+// Changing the default visibility to hidden has the advantage of significantly
+// reducing the code size and load times, as well as letting the optimizer
+// produce better code.
```

Comment:
Please make *only* automated changes in this commit. Anything else that was done manually should please be in a separate commit.